projects/templates/src/lib/components/po-page-dynamic-edit/po-page-dynamic-edit.component.ts

```diff
@@ -316,6 +318,53 @@ export class PoPageDynamicEditComponent implements OnInit, OnDestroy {
     return { ...this._actions };
   }
 
+  /**
+   * @optional
+   *
+   * @description
+   *
+   * Objeto com as literais usadas no `po-page-dynamic-edit`.
+   *
+   * É possivel customizar passando um objeto com todas as literais disponíveis
+   * ou passando apenas as literais que deseja customizar
+   *
+   * ```
+   *  const customLiterals: PoPageDynamicSearchLiterals = {
+   *    detailActionNew: 'Incluir',
+   *    pageActionCancel: 'Descartar',
+   *    pageActionSave: 'Gravar',
+   *    pageActionSaveNew: 'Gravar e incluir',
```

Comment:
Acho legal colocar todas as opções aqui pois facilita para o dev.
```suggestion
   *    detailActionNew: 'Incluir',
   *    pageActionCancel: 'Descartar',
   *    pageActionSave: 'Gravar',
   *    pageActionSaveNew: 'Gravar e incluir',
   *    registerNotFound: 'Nenhum registro encontrado.',
   *    saveNotificationSuccessSave: 'Item salvo com sucesso.',
   *    saveNotificationSuccessUpdate: 'Item atualizado com sucesso.',
   *    saveNotificationWarning: 'Necessário preencher o formulário corretamente.'
```